Editor/Scripts/CodeGenerators/RGActionCodeGenerator.cs

```diff
@@ -16,6 +16,17 @@ public class RGActionCodeGenerator
         [MenuItem("Regression Games/Generate Scripts")]
         private static void GenerateRGScripts()
         {
+            // remove old 'RGScripts' folder that is no longer used
+            string dataPath = Application.dataPath;
+            string directoryToDelete = Path.Combine(dataPath, "RGScripts").Replace("\\", "/");
+
+            if (Directory.Exists(directoryToDelete))
+            {
+                Directory.Delete(directoryToDelete, true);
+                File.Delete(directoryToDelete + ".meta");
+                AssetDatabase.Refresh();
+            }
```

Comment:
We can probably make a ticket to remove this at some point? I think we should get in the habit of cleaning up old cold that becomes outdated. Thoughts on that process?

Reply:
I'm almost inclined to put a TODO here without a task # that lists a remove after X date indicator on it.  If someone is in there changing and sees it great.

I'm hesitant to suggest we create tasks for cleanup of single lines like this as historically all I've ever seen that do is add tasks to the backlog that never get prioritized :/

Reply:
That's fine with me! I think a TODO is fine just so we know that this code can be removed in the future.

Reply:
The other approach, if you wanted to do a ticket, is to make a ticket and explicitly put it into the cycle that is 2 cycles ahead from now, but I am fine with either way.

Reply:
todo added